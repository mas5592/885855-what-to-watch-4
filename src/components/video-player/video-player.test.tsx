import * as React from 'react';
import * as renderer from 'react-test-renderer';
import VideoPlayer from './video-player';

export const mock = {
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  image: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
};

describe(`VideoPlayer`, () => {
  it(`Should render correctly component`, () => {
    const {src, image} = mock;
    const tree = renderer
      .create(
          <VideoPlayer
            isPlaying = {true}
            source={src}
            poster={image}
          />, {
            createNodeMock: () => {
              return {};
            }
          }).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

