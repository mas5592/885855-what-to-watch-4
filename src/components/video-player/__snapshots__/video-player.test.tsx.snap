// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer Should render correctly component 1`] = `
<video
  className="player__video"
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
/>
`;
